site/app/templates/Vue.twig

```diff
@@ -0,0 +1,8 @@
+<script> 
+    window.addEventListener('load', () => {
+        window.submitty.render('[data-vuemount]', "{{ page }}",  {{ args | json_encode() | raw() }});
+    })
```

Comment:
You could use jQuery to do this instead.

Reply:
Addressed in https://github.com/Submitty/Submitty/pull/10715/commits/f08c60b4e8100eaf8d93c6ee67e8ad38bed300e9